pkg/repository/v1/workflow.go

```diff
@@ -485,12 +488,22 @@ func (r *workflowRepository) createWorkflowVersionTxs(ctx context.Context, tx sq
 	}
 
 	for _, eventTrigger := range opts.EventTriggers {
+		expression := pgtype.Text{
+			String: "",
+			Valid:  false,
+		}
```

Comment:
is this the right way to do this?

Reply:
What you've initialized is the zero value of `pgtype.Text` (importantly because this _isn't_ a pointer, and the zero value of bools is `false` and strings is the empty string), so it's not necessary to even instantiate anything here since the zero value is the default value